invenio_preservation_sync/utils.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Utils for Preservation Sync module."""
+
+from flask import current_app, g, url_for
+
+from invenio_preservation_sync.models import PreservationStatus
+
+from .proxies import current_preservation_sync_service as service
+
+
+def preservation_info_render(record):
+    """Render the preservation info."""
+    permissions = record.has_permissions_to(["manage"])
+    can_manage = permissions.get("can_manage", False)
+
+    pid = record._record.pid.pid_value
+    result = service.read(g.identity, pid, latest=True).data
+
+    if not result or (
+        result["status"] != PreservationStatus.PRESERVED and not can_manage
+    ):
+        return []
+
+    title = current_app.config.get(
+        "PRESERVATION_SYNC_UI_TITLE", "Preservation Platform"
+    )
+    url = current_app.config.get("PRESERVATION_SYNC_UI_LINK", result["uri"])
+    if can_manage and not current_app.config.get(
```

Comment:
What is the need of the extra config `PRESERVATION_SYNC_UI_MANAGER_LINK_OVERRIDE`? In which would I change the URL?

Reply:
The idea was to cover the CDS-RDM case:
- For regular users we show a general link about preservation at CERN (hence the `PRESERVATION_SYNC_UI_LINK` config to override)
- For admins we could use the link that actually points to the preservation in some form

Is this not needed? Should admins also see the overridden link?

Reply:
Oh understood. I was meaning something slightly different.
A link to an explanation page should always be there, with a little `info` icon for example, see example:
![Screenshot 2024-10-31 at 14 20 36](https://github.com/user-attachments/assets/aa35d78e-9cca-46e1-9b24-8b9d0e133ac5)
(we should find a better icon!!!)

Making the link clickable and go to the DM platform should only visible to sys admins.